zulipterminal/ui_tools/views.py

```diff
@@ -1329,6 +1330,14 @@ def __init__(self, controller: Any, stream_id: int) -> None:
             )
         ]
 
+        if stream.get("stream_post_policy") is None:
```

Comment:
Where we're using default values, it is clearer if you use `"text" in some_dict`, as that indicates we're testing for membership. The cost is similar, but "fetch and return an error value, which we test against" is not as obvious as explicitly using `in`.